patches/net/minecraft/client/renderer/Sheets.java.patch

```diff
@@ -3,29 +3,36 @@
 @@ -172,11 +_,13 @@
      }
  
-     private static Material createSignMaterial(WoodType p_173386_) {
+     public static Material createSignMaterial(WoodType p_173386_) {
 -        return new Material(SIGN_SHEET, new ResourceLocation("entity/signs/" + p_173386_.name()));
 +        ResourceLocation location = new ResourceLocation(p_173386_.name());
 +        return new Material(SIGN_SHEET, new ResourceLocation(location.getNamespace(), "entity/signs/" + location.getPath()));
      }
  
-     private static Material createHangingSignMaterial(WoodType p_251735_) {
+     public static Material createHangingSignMaterial(WoodType p_251735_) {
```

Comment:
Why are these patched to be public while also using an AT to make them public... Remove these in favor of the AT entries

Reply:
as per gigaherz in neocord
> the process always needs to be add ATs -> setup -> edit code -> build and generate patches

if done in the wrong order, patches will generate with ATs applied, which is erroneous

Reply:
still results in this happening.